app/models/upload.rb

```diff
@@ -3,6 +3,10 @@
 require "digest/sha1"
 
 class Upload < ActiveRecord::Base
+  self.ignored_columns = [
+    "verified" # TODO(2020-12-10): remove
```

Comment:
We should mark this column as readonly first to prevent any writes going to it.

Reply:
For this you mean `Migration::ColumnDropper.mark_readonly(:uploads, :verified)` yes?

Reply:
@tgxworld

Reply:
Yup :)